src/ProjectNeon/Assets/Data/Templates/Member.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Member : Target
+{
+    [SerializeField] private Character character;
+    [SerializeField] private Deck deck;
+    [SerializeField] public int hp;

```

Comment:
Put a puzzle here, so that other classes don't directly mutate HP. We eventually want to use accessors so that we can setup reactive bindings. 

This will be very important for trigger UI bindings, Death triggers, visual effects and so forth.